README.md

```diff
@@ -345,40 +352,45 @@ serve({ config }).then((server) => {
 
 #### build-started
 
-Arguments:  
-  [`Compiler`](https://webpack.js.org/api/node/#compiler-instance) _compiler_
+Arguments:
+
```

Comment:
> I accidentaly ran prettier on the readme.md but the changes seemed good so I just stuck with it. (sorry for the random changes is what I'm trying to say).

Maybe (especially since i mentioned this in my pull request) there is a nicer way to phrase this and not spam me with the same message about something that is not my personal preference at all (as a matter of fact I didn't realize it happened until I started writing the pr message).